CMakeLists.txt

```diff
@@ -526,4 +526,31 @@ else ()
                         "${GTEST_SOURCE_DIR}/googletest/include")
     target_link_libraries(draco_tests draco gtest)
   endif ()
+
+  # Headers installation
+  file ( GLOB_RECURSE draco_headers RELATIVE ${CMAKE_SOURCE_DIR} "*.h" )
+  foreach (filename ${draco_headers})
+    get_filename_component ( file_directory ${filename} DIRECTORY )
```

Comment:
style nit, please collapse unnecessary horizontal white space, this:
get_filename_component ( file_directory ${filename} DIRECTORY )

should be:
get_filename_component(file_directory ${filename} DIRECTORY)